Jenkinsfile

```diff
@@ -77,9 +77,9 @@ pipeline {
 
     stage('Submit Coverage Report'){
       steps{
-        archiveArtifacts artifacts: "coverage/.resultset.json", fingerprint: false
-        archiveArtifacts artifacts: "ci/authn-k8s/output/simplecov-resultset-authnk8s-gke.json", fingerprint: false
         sh 'ci/submit-coverage'
+        archiveArtifacts artifacts: "coverage/.resultset*.json", fingerprint: false
+        archiveArtifacts artifacts: "ci/authn-k8s/output/simplecov-resultset-authnk8s-gke.json", fingerprint: false
```

Comment:
We put these lines before `ci/submit-coverage` previously so that we could download and examine the artifacts from Jenkins in the event of an error. Do we no longer want that option?

Reply:
I moved it so that pre and post merge arrtifacts could be downloaded. I failed to realise that if there was an error in the submit-coverage script, the archiveArtifacts step wouldn't be reached. I'll add an exception handler.